docs/reference/run.md

```diff
@@ -463,6 +463,9 @@ the container exits**, you can add the `--rm` flag:
 
     --rm=false: Automatically remove the container when it exits (incompatible with -d)
 
+> Note that when you set `--rm` flag, Docker removes the volumes associated 
```

Comment:
Hm, reading back, think I should've suggested "if you set". Guess it's good enough for now :relaxed: